src/Microsoft.Azure.WebJobs.Logging/WebJobs.Logging.csproj

```diff
@@ -1,150 +1,29 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I *think* this Logging project is mostly for the Dashboard. I wonder if we can get rid of it? Probably not necessary in this first wave, but something to look into.